apps/factory/src/components/prompt_output_log.tsx

```diff
@@ -37,10 +37,12 @@ const PromotOutputLog: React.FC<PromotOutputLogProps> = ({ pl }) => {
 
   useEffect(() => {
     if (typeof window !== "undefined") {
-      const completePath = `${window.location.href}/logs/${pl?.id}`;
+      const completePath = `${
+        window.location.href.split("?")[0]
+      }/logs/${pl?.id}`;
       setPromptLogUrl(completePath);
```

Comment:
@LovedeepSingh19 why need to add this kind of hacks ?

Reply:
if there are multiple versions in editor mode then this is the URL:-
http://localhost:3000/dashboard/prompts/a73fa4bf-6a86-40ea-8880-b79ec8bddb1b?ptid=3aee7ca5-bcee-4b36-83a1-b65137f7b8c0

here "?pid=" is used for the version id which was causing that issue while redirecting